modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -456,7 +458,9 @@ export default class LargeVideoManager {
         if (presenceLabelContainer.length) {
             ReactDOM.render(
                 <Provider store = { APP.store }>
-                    <PresenceLabel participantID = { id } />
+                    <I18nextProvider i18n = { i18next }>
```

Comment:
Isn't there a central place to put this I18nextProvider component into so then all other pages inherit it?

Reply:
I suppose this place is AbstractApp.js and the provider is already there. 

Since this is separate ReactDOM.render I don't think we can define it on different place(AbstractApp.js). But once all the code is reactified the one defined in AbstractApp should be enough.